### PR TITLE
Added comment reporting, fixed issue with GraphQL subscriptions

### PR DESCRIPTION
**Pull request by dabit3**

### CODE REVIEW

**Comment by robinheinze** [comment]
Thanks, Nader! I'll take a look at this today!

File: app/screens/talk-details/talk-details-screen.tsx [hunk 1]
```diff
@@ -98,14 +98,22 @@ const TAB_HOLDER: ViewStyle = { flex: 1 }
...
+  paddingTop: 20,
+  paddingHorizontal: 20,
+  borderTopWidth: 1,
+  borderTopColor: "rgba(255, 255, 255, .1)",
+}
 const CREATED_BY = { color: "white", fontWeight: "600" }
 const CREATED_AT = { color: "rgba(255, 255, 255, .5)", fontSize: 11, marginLeft: 8, marginTop: 3 }
 const INPUT_CONTAINER = { bottom: 0, position: "absolute", left: 0 }
 const MESSAGE_INPUT = { backgroundColor: "white", height: 50, paddingHorizontal: 8 }
+const REPORT = { marginTop: 5, color: "green" }
```
Comment (robinheinze) [inline app/screens/talk-details/talk-details-screen.tsx]: Can we make this text a red-ish color (I think `palette.angry` or `palette.crimson` would work well), and just a bit smaller so it's not the same size as the comment
```suggestion
const REPORT = { marginTop: 5, color: palette.angry, fontSize: 11 }
```

Comment (robinheinze) [reply app/screens/talk-details/talk-details-screen.tsx]: Screen Shot 2019-06-18 at 12 29 31 PM

**Comment by robinheinze** [comment]
Yeah, ideal case, this would be an action sheet under ellipses along with other actions like the ability to delete and edit your own messages. But given the time constraints, this is fine.

**Comment by dabit3** [comment]
Thanks for the suggestion @robinheinze, just added those changes.

**Comment by robinheinze** [comment]
Merged! Fixes #95  and #78

**Pull request merged**